ontology.ttl

```diff
@@ -968,6 +968,23 @@ oe:dos_heeftAlsBeslisser a owl:ObjectProperty;
     taken by the decision maker.
     """@en.
 
+oe:dos_heeftAlsMedewerker a owl:ObjectProperty;
+    rdfs:isDefinedBy <https://id.erfgoed.net/vocab/ontology>;
+    rdfs:label "heeft als medewerker"@nl;
+    rdfs:label "has as contributor"@en;
```

Comment:
is medewerker niet eerder collaborator :thinking:

Reply:
goh ja,
![image](https://user-images.githubusercontent.com/1926276/231746820-4c08b691-8876-4047-8ef7-13bb3b79a37d.png)

Ik vond contributor wel passend klinken maar maakt me niet zoveel uit, ik maak er collaborator van